aws4authwebsocket/transport.py

```diff
@@ -0,0 +1,299 @@
+from gremlin_python.driver.transport import AbstractBaseTransport
+import mocket.mocket
+import mocket.mockhttp
+from overrides import overrides
+import os
+from requests import PreparedRequest
+from requests_aws4auth import AWS4Auth
+import threading
+from typing import Any, Callable, List, Mapping, Optional, Tuple, TypeVar, Union
+from urllib.parse import urlparse
+from websocket import create_connection
+
+
+__all__ = ["WebsocketClientTransport", "Aws4AuthWebsocketTransport"]
+
+
+def monkey_patch_mocket() -> None:
```

Comment:
the file seems to be related only for the new proxy, do you think we could move it under proxy folder like https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/proxy/statsd_utilities.py instead of upper folder in metadata?